doc/application_plugins/index.rst

```diff
@@ -23,9 +23,9 @@ Pluggable Authentication Module
 
 .. index:: offline, PAM
 
-The PAM module of privacyIDEA directly communicates with the privacyIDEA
-server via the API. The PAM module also supports offline authentication. In
-this case you need to configure an offline machine application. (See
+The `PAM module of privacyIDEA <https://github.com/privacyidea/pam_python>`_ directly
+communicates with the privacyIDEA server via the API. The PAM module also supports offline
+authentication. In this case you need to configure an offline machine application. (See
 :ref:`application_offline`)
 
 You can install the PAM module with a ready made Debian package for Ubuntu or
```

Comment:
So we also should remote "ready made Debian package"?

Reply:
huh, missed that, was looking for ubuntu :-) will fix.

Reply:
Done in e4a50d61